content/efcore/getting-started/samples/manipulating-entities/updating-entities/Program.cs

```diff
@@ -7,19 +7,16 @@ public static void Main()
     {
         using (var context = new LibraryContext())
         {
-            var book = context.Books.First();
+            var book = context.Books
+				.Single(b => b.Title == "Mrs Dalloway"); 
 
-            Console.WriteLine(String.Format("Original Book - {0}: {1}", book.Id, book.Title));
-
-            book.Title = "Frankenstein: or, The Modern Prometheus";
-            context.SaveChanges();
-        }
-
-        using (var context = new LibraryContext()) {
-            var updatedBook = context.Books
-                .Single(b => b.Title.Contains("Frankenstein"));
-
-            Console.WriteLine(String.Format("Updated Book - {0}: {1}", updatedBook.Id, updatedBook.Title));
+            Console.WriteLine("Original Book -\n Id: {0}\n Title: {1}\n Genre: {2}\n Publication Year: {3}", book.Id, book.Title, book.Genre, book.PublicationYear);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.